src/ios/RNInAppBrowser.podspec

```diff
@@ -1,13 +1,15 @@
-# Keep in sync with package.json
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, '..', '..', 'package.json')))
```

Comment:
Should go up one parent folder less because the `ios` folder will be placed at the same level as `package.json` in the tarball uploaded to NPM.